wiki/Hit_object/Fruit/en.md

```diff
@@ -0,0 +1,11 @@
+---
+stub: true
+---
+
+# Fruit
+
+*Fruits* are large coloured [hit objects](/wiki/Hit_object) present in [osu!catch](/wiki/Game_mode/osu!catch) [beatmaps](/wiki/Beatmap). A [hyperfruit](/wiki/Hit_object/Hyperfruit) will be generated on a fruit once the next fruit or [drop](/wiki/Hit_object/Juice_stream#drop) is uncatchable by normal dashing.
+
+Successfully catching a fruit will earn the player [score](/wiki/Score), increase the [combo](/wiki/Beatmapping/Combo) by 1, provide a boost to the [health bar](/wiki/Glossary/Health_bar), and is treated as 300 in the result screen. If the player fails to catch a fruit it will result in a [combo](/wiki/Beatmapping/Combo) break and a loss in [health](/wiki/Beatmapping/Health).
```

Comment:
```suggestion
Successfully catching a fruit will earn the player [score](/wiki/Score), increase the [combo](/wiki/Beatmapping/Combo) by 1, provide a boost to the [health bar](/wiki/Glossary/Health_bar), and is treated as 300 in the result screen. If the player fails to catch a fruit it will result in a [combobreak](/wiki/Glossary/Combobreak) and a loss in [health](/wiki/Beatmapping/Health).
```